### PR TITLE
Python: Promote ClickHouse SQL models

### PR DESCRIPTION
**Pull request by RasmusWL**

### CODE REVIEW

**Comment by RasmusWL** [comment]
seems like stripping leading whitespace sneaked into my merge commit... I don't feel too sorry about that :smile:

File: python/ql/test/library-tests/frameworks/aioch/ConceptsTest.ql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import python
+import experimental.meta.ConceptsTest
```
Comment (yoff) [inline python/ql/test/library-tests/frameworks/aioch/ConceptsTest.ql:2]: Perhaps we should also promote this at some point..

**Review by yoff — APPROVED** [review]
LGTM, I think splitting out `aioch` works well.

File: python/ql/src/experimental/semmle/python/frameworks/ClickhouseDriver.qll [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        result = classRef.getASubclass*()
+      )
+    }
+
+    /** Gets a reference to an instance of `clickhouse_driver.Client` or any subclass. */
+    API::Node instance() { result = subclassRef().getReturn() }
+  }
+
+  /** `clickhouse_driver.Client` execute method names */
+  string getExecuteMethodName() { result in ["execute_with_progress", "execute", "execute_iter"] }
```
[the remark this answers is not in the document]
Comment (RasmusWL) [reply python/ql/src/experimental/semmle/python/frameworks/ClickhouseDriver.qll]: PR welcome :blush:

**Pull request merged**